packages/example-app-base/src/components/ApiReferencesTable/index.ts

```diff
@@ -1,3 +1,3 @@
-export * from './ApiReferencesTable';
+export { ApiReferencesTable, IApiReferencesTableState, gapTokens } from './ApiReferencesTable';
```

Comment:
What's the reason for changing this export?

Reply:
ApiReferencesTable.tsx re-exports IApiReferencesTableProps (from the types file) for some reason. I think this got flagged by a no duplicate exports rule at some point, though I might have turned that rule off due to issues in other places or bad perf.